app/src/main/res/layout/activity_send_request.xml

```diff
@@ -48,14 +48,26 @@
             android:id="@+id/tvRequestEndDate"
             android:layout_width="0dp"
             android:layout_height="wrap_content"
-            android:layout_marginEnd="20dp"
```

Comment:
Why not have the disable attribute here instead of disabling programmatically? Just an idea.

Reply:
Yaa , we can disable here also but I prefer programmatically.